### PR TITLE
change to log.go v2

### PR DESCRIPTION
**Pull request by wavemechanics**
### What

Change to log.go v2.

### How to review

make test

### Who can review

Anybody familiar with log.go v2.

### CODE REVIEW

File: example/example.go [hunk 1]
```diff
@@ -19,23 +19,23 @@ func main() {
 
 	// In production no tokens should be logged
 	logData := log.Data{"address": devAddress, "token": token}
-	log.Event(nil, "Created vault client", log.INFO, logData)
+	log.Info(nil, "Created vault client", logData)
```
Comment (cadmiumcat) [inline example/example.go]: I'd take this opportunity to change the `nil` context to `context.Background`. I think we're not meant to pass `nil` context https://pkg.go.dev/context 

I realise this is just an example, so I won't block the PR!

**Review by cadmiumcat — DISMISSED** [review]
👍

File: example/example.go (hunk 1 above)
Comment (wavemechanics) [reply example/example.go]: Yeah, it's better to show correct usage in an example anyway. Fixed.

**Review by cadmiumcat — DISMISSED** [review]
🥳 nice

**Review by cadmiumcat — DISMISSED** [review]

**Review by cadmiumcat — APPROVED** [review]

**Pull request merged**